ddtrace/contrib/logging/__init__.py

```diff
@@ -40,7 +45,9 @@ def hello():
     from ddtrace import tracer
 
     FORMAT = ('%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] '
-              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] '
+              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s'
```

Comment:
must be `[{key:value} dd.trace_id={id} dd.span_id={id}]`, `dd.trace_id=0 dd.span_id=0` must always be last,

Reply:
oh.. we should document this too then..